### ISSUE TITLE
Train a first-pass stochastic model (multivariate Gaussian)

### ISSUE THREAD

**Issue by jiwoncpark** [LSSTDESC/derp#9]
Once we see that the features in our training data allow for okay training, we implement a simple type of stochastic model parameterized as a multivariate Gaussian, where each dimension belongs to a feature in the output space (the space of object-catalog features).

Call the input (truth-catalog) features `x` and the output (object-catalog) features `y`. The central challenge of this task is defining a loss function that captures the distance between the current network distribution `q(y|x; theta)` where `theta` refers to the Gaussian parameters and the target distribution `p(y|x)`.
